lib/gutenberg/blocks/embeds-block-component.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+import { By } from 'selenium-webdriver';
+import * as driverHelper from '../../driver-helper';
+import GutenbergBlockComponent from './gutenberg-block-component';
+
+export default class EmbedsBlockComponent extends GutenbergBlockComponent {
+	constructor( driver, blockID ) {
+		super( driver, blockID );
+	}
+
+	async embedUrl( url ) {
+		await driverHelper.setWhenSettable(
+			this.driver,
+			By.css( `${ this.blockID } .components-placeholder__input` ),
```

Comment:
`this.blockID` ensures that the correct block is populated with URL and that correct 'Embed` button is clicked. Because of this we can embed several urls in the same test.